### ISSUE TITLE
Broken ellipse after recompiling

### ISSUE THREAD

**Issue by torkve** [jgraph/draw.io#17]
I tried to recompile the app with `ant -buildfile etc/build/build.xml all` and it made all the shapes look a bit different from clean checkout, and made ellipse broken (it became rect) at all — see attached image.
Browser didn't matter, I tried Firefox 49.0 and Chromium 53.0.2785.143.
The problem was reproduced on my Ubuntu 16.10, as well as with clean Docker image with Ubuntu 14.04.
```docker
FROM ubuntu:trusty
ADD . /
WORKDIR war
RUN apt-get update -qq
RUN apt-get -y --force-yes install python3.4 patch ant openjdk-7-jdk
RUN ant -buildfile ../etc/build/build.xml all
RUN echo $'import http.server, socket, sys\n\
class V6(http.server.HTTPServer):\n\
  address_family = socket.AF_INET6\n\
http.server.test(ServerClass=V6, \n\
                 HandlerClass=http.server.SimpleHTTPRequestHandler, \n\
                 bind="::", \n\
                 port=int(sys.argv[1]))' > /httpserver.py
CMD /usr/bin/python3.4 /httpserver.py 80
```

![drawio-ellipse](https://cloud.githubusercontent.com/assets/770110/19530080/62e453bc-9643-11e6-8760-563c07e53b78.png)

**Comment by davidjgraph** [comment]
Any errors on the console?

**Comment by torkve** [comment]
No errors, but a number of warnings: http://pastebin.com/Ar0f17Bu

**Comment by davidjgraph** [comment]
I mean the browser console when you run the app. If you have dev=1 set you need mxGraph correctly available as described in https://github.com/jgraph/draw.io/issues/16.

**Comment by torkve** [comment]
No, I don't use dev=1.
The only difference Firefox reports in recompiled app:
```
14:23:12.426 element not found 1 localhost:32787:1:1
14:23:12.451 Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help, check http://xhr.spec.whatwg.org/ 1 app.min.js:324:226
```
Unfortunately I cannot find out, what "element" is written about, since 1:1 is definitely wrong row/column pointer.

Chromium reported only about application cache loading progress, from which I found that recompiled cache.manifest is empty:
```
CACHE MANIFEST

# THIS FILE WAS GENERATED. DO NOT MODIFY!
# 10/19/2016 03:48 PM
```

**Comment by torkve** [comment]
As far as I found, restoring original cache.manifiest doesn't help.

Practically the only thing that changed is theme in app.min.js. To find out what changes, I processed the repository one (old) and recompiled one (new) app.min.js through js-beautifuer and looked at diff.
I found that the only thing changed is line
```javascript
Graph.prototype.defaultThemes[Graph.prototype.defaultThemeName] = mxUtils.parseXml('%long xml%').documentElement;
```
after recompilation the inner xml literal is replaced with empty string. So it probably somehow collides with [templates compilation](https://github.com/jgraph/draw.io/blob/master/etc/build/build.xml#L50).
